packages/contracts/.vscode/launch.json

```diff
@@ -25,7 +25,7 @@
             "request": "launch",
             "name": "script: deployRoundImplementation",
             "runtimeExecutable": "yarn",
-            "args": ["goerli-deploy-round"],
```

Comment:
This is gonna be surprising but I do not use VSCode so hard for me to determine what these do. 😆

Reply:
Haha i came across on some random twitter thread. 
This PR should explain what it does but in short -> let's us put in debug points. What editor do you atm ?
https://github.com/gitcoinco/grants-round/pull/23

Reply:
Sublime Text 😎